jcl/src/java.base/share/classes/openj9/tools/attach/diagnostics/base/DiagnosticUtils.java

```diff
@@ -212,14 +229,49 @@ private static DiagnosticProperties getThreadInfo(String diagnosticCommand) {
 		return result;
 	}
 
+	private static DiagnosticProperties doDump(String diagnosticCommand) {
+		DiagnosticProperties result = null;
+		String[] parts = diagnosticCommand.split(DIAGNOSTICS_OPTION_SEPARATOR);
```

Comment:
Is there a test against a `diagnosticCommand` with `DIAGNOSTICS_OPTION_SEPARATOR`?

Reply:
Yes,  for example `testThreadPrint()`.

Reply:
Since `DIAGNOSTICS_OPTION_SEPARATOR = ","`, I assume there is a `,` within the test command options however  It doesn't seem obvious within `testThreadPrint()`.
In addition, `testThreadPrint()` is not part of this PR, does the newly added test provide code coverage for the code path when `DIAGNOSTICS_OPTION_SEPARATOR` presents?

Reply:
The separator is inserted by `DiagnosticUtils.makeJcmdCommand()`. This is exercised by other tests, as well as this test: the separator is used to append the file name.  Running `jcmd <target> Dump.java myJavaDump` with logging enabled, the target receives `doCommand ATTACH_DIAGNOSTICS:Dump.java,myJavaDump`.